themes/custom/droopler_theme/scss/components/_d_commerce.scss

```diff
@@ -1,5 +1,111 @@
-body {
-  h1 {
-    font-size: 2rem;
+@mixin btn-slider-navigation-next {
+  z-index: 1;
+  height: 2.5rem;
+  width: 2.5rem;
+  background: $white;
+  opacity: 0.95;
+  &:hover {
+    opacity: 1;
+  }
+  &::before {
+    content: " ";
+    width: 0.8rem;
+    height: 0.8rem;
+    border-right: 0.125rem solid;
+    border-top: 0.125rem solid;
+    border-color: $color-troilus;
+    display: inline-block;
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+    z-index: 1;
+  }
+}
+
+@mixin btn-slider-navigation-prev {
+  @include btn-slider-navigation-next;
+  &::before {
+    -webkit-transform: rotate(-135deg);
+    -ms-transform: rotate(-135deg);
+    transform: rotate(-135deg);
+  }
+}
+
+.view-commerce-product-images {
+  overflow: hidden;
+
+  .slick-slide {
+    .field--type-image {
+      margin-right: 0;
+      img {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .slider-main {
+    margin: 0 0 1.5rem 0;
+    .img-container {
+      position: relative;
+      overflow: hidden;
+    }
+  }
+
+  .slider-nav {
+    .slick-track,
+    .slick-track-const {
+      margin-left: 0;
+      margin-right: 0;
+    }
+    .slick-list {
+      margin-left: -0.7rem;
+      margin-right: -0.7rem;
+      padding: 0 !important;
+      img:hover {
+        cursor: pointer;
+      }
+      .slick-track-const {
+        display: flex;
+        justify-content: center;
+        width: 100% !important;
+        transform: none !important;
+      }
+      .slick-slide {
+        &.slick-current {
+          outline: 0;
+        }
+      }
+    }
+  }
+  .slick-prev {
+    @include btn-slider-navigation-prev();
+    padding-left: 0.3rem;
+    margin-left: 1rem;
+    left: 0;
+  }
+  .slick-next {
+    @include btn-slider-navigation-next;
+    padding-right: 0.3rem;
+    margin-right: 1rem;
+    right: 0;
+  }
+}
+
+.product__content {
+  .section-data {
+    padding-top: 15px;
+
+    .field__label {
+      font-size: 1.2rem;
+      font-weight: 700;
+      line-height: 1.2rem;
```

Comment:
@grzegorz-pietrzak-droptica  jak to teraz na droopler ? kiedyś trzeba było wrzucać takie rzeczy jak font sizy do zmiennych itp, teraz tez trzeba czy jak kto woli ? :)

Reply:
Zależy jak bardzo chcemy mieć możliwość customizacji, na pewno trzeba robić to przy kolorach, font sizy podane w REM raczej mogą zostać